<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookEntry</class>
 <widget class="QDialog" name="BookEntry">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>696</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>660</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string>The title of the work.</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPlace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Place:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPublisher">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Publisher:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblYear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPageCnt">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pages</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPrice">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAJBnum">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>59</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AJB Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblItem">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAuthor1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAuthor2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAuthor3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author 3</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="teReviews">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>701</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblReview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reviews:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="teComments">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>701</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btnPrevious">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show previous record.</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::LeftArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnNext">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show next record.</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="arrowType">
    <enum>Qt::RightArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="btnGetNextFile">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Get File ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRecordCount">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Record   0 of    0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>291</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookEntry</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookEntry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
